pkg/virt-controller/services/net_binding.go

```diff
@@ -44,33 +45,56 @@ func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.Kube
 			if !exist {
 				return nil, fmt.Errorf("couldn't find configuration for network bindining: %s", iface.Binding.Name)
 			}
+		} else if iface.Slirp != nil {
+			pluginSidecars = append(pluginSidecars, SlirpNetBindingPluginSidecar(vmi, config, recorder))
```

Comment:
Before we had this code block creating a `bindingByName` pool and only later appending to the sidecars pool. Now we have a mix that makes me dizzy.

Just do another for-loop and do not mix things together, it makes it hard to follow this and hard to move later.

Reply:
Done